src/convexhull.jl

```diff
@@ -0,0 +1,36 @@
+"""
+    upperhull(pset)
+
+Upper part of the convex hull of a given set of 2D points `pset`. 
+The upperhull is computed by the "Graham Scan" algorithm.
```

Comment:
Can you add a reference to the algorithm like in other doc strings?

Reply:
Is this [wikipedia page](https://en.wikipedia.org/wiki/Graham_scan) good for this?

Reply:
In wikipedia the algorithm is from bottom to top, whereas I'm implementing it  from left to right.

Reply:
This comment seems fitting to our discussion (also form wikipedia) : 

"The same basic idea works also if the input is sorted on x-coordinate instead of angle, and the hull is computed in two steps producing the upper and the lower parts of the hull respectively. This modification was devised by A. M. Andrew[2] and is known as Andrew's Monotone Chain Algorithm. It has the same basic properties as Graham's scan."

Reply:
Nice. Please cite the exact algorithm that is being implemented and add this excerpt from Wikipedia to the docstring so that future readers understand what is going on.